selenium/codenvy-selenium-test/src/test/java/com/codenvy/selenium/factory/AuthenticateAndAcceptFactoryThroughGitHubOAuthTest.java

```diff
@@ -107,4 +111,43 @@ public void loginThroughGitHubOAuthAndAcceptFactory() throws Exception {
     projectExplorer.waitProjectExplorer();
     notificationsPopupPanel.waitExpectedMessageOnProgressPanelAndClosed("Project Spring imported");
   }
+
+  private void deleteAllWorkspaces(List<String> workspaces) {
+    workspaces.forEach(
+        wsName -> {
+          seleniumWebDriver.get(getDashboardWorkspaceUrl());
+          dashboardWorkspace.waitToolbarTitleName(WORKSPACE_TOOLBAR);
+          dashboardWorkspace.waitListWorkspacesOnDashboard();
+          dashboardWorkspace.selectWorkspaceItemName(wsName);
+          dashboardWorkspace.waitToolbarTitleName(Arrays.asList(wsName.split("/")).get(1));
+          dashboardWorkspace.clickOnDeleteWorkspace();
+          dashboardWorkspace.clickOnDeleteDialogButton();
+          dashboardWorkspace.waitToolbarTitleName(WORKSPACE_TOOLBAR);
```

Comment:
We need to ensure that workspace was completely removed.

Reply:
Yes, you are right, good idea

Reply:
There is notification popup at the top right corner appears after workspace has been successfully removed.